doc/sphinx/source/project/release-notes.rst

```diff
@@ -1,41 +1,61 @@
-.. release-notes:
+.. ngraph/release-notes:
 
 Release Notes
 #############
 
-This is |release|.
+The latest |version| download below.
 
+* `Format .zip`_ 
+* `Format tar.gz`_ 
 
+See also: https://github.com/NervanaSystems/ngraph/releases for previous versions. 
 
 
+CHANGELOG |release|
+-------------------
 
-CHANGELOG
-=========
+nGraph v0.17.0-rc.0
+~~~~~~~~~~~~~~~~~~~
 
-(Last updated September 2018)
++ Allow negative padding in more places
++ Add code generation for some quantized ops
++ Preliminary dynamic shape support
++ initial distributed ops
```

Comment:
I just added an API section to the release notes:
API Changes
* Pad op takes CoordinateDiff instead of Shape pad values to allow for negative padding.

Reply:
Done